### PR TITLE
Develop a living style guide

### PR DESCRIPTION
**Pull request by sarahmonster**
In-progress; developing a style guide for the site. So far it's mostly just a half-done logo, some needs-refining type styles, and a colour scheme:

screenshot 2016-11-29 15 32 34

Will require quite a bit more work before it's ready to merge.

Style guide itself is accessible at <your-localhost>/wp-content/themes/strikebase/assets/style-guide/

### CODE REVIEW

**Comment by michiecat** [comment]
I dig the colors and logo, and look forward to seeing how this develops. :)

**Comment by sarahmonster** [comment]
I think this is now ready to go. The style guide isn't 100% done, but I'd like to add more elements to it as we go along building up a library of design patterns, and this is a good structure to start with.

You can see the style guide in your browser via the theme's directory: `/wp-content/themes/strikebase/assets/style-guide/`. The URL will differ slightly depending on your local development environment; for me it's http://strikebase.dev/wp-content/themes/strikebase/assets/style-guide/

I've totally revamped the style guide so it's way more comprehensive. Here's a quick couple of screenshots:

strikebase style guide 1


strikebase style guide

Working on the style guide also allowed me to audit some of the existing styles and tweak them, since I can see them all in one place. So I've desaturated the colours a bit, I've added some better headings hierarchy, etc. This will be a continual work in progress as the style guide is updated along with page layouts.

Since building a style guide largely goes hand-in-hand with the actual styling of elements (at least the way I do it), this PR also introduces some actual designed elements, notably on the individual people/projects pages:

sarah semark strikebase


recording conversations strikebase 1

Obviously not everything is at 100% yet, but this lays the groundwork, and I can work in a more modular way with (hopefully) smaller PRs after this. 

A quick review would be lovely—this is merge-ready.

File: archive-person.php [hunk 1]
```diff
@@ -11,23 +11,21 @@
...
-				<thead>
-					<th><?php esc_html_e( 'Name', 'strikebase' ); ?></th>
-					<th><?php esc_html_e( 'Organization', 'strikebase' ); ?></th>
-					<th><?php esc_html_e( 'Projects', 'strikebase' ); ?></th>
-				</thead>
+			<div class="strikebase-filter-and-sort">
+				<?php strikebase_filters( 'person-type', 'Type' ); ?>
+			</div>
+
+			<div class="strikebase-people-list">
```
Comment (danielwrobert) [inline archive-person.php]: I'll need to adjust the components in the React version to reflect any changes that you made in the markup. This is to be expected and not a bad thing. Just noting here as I saw the `table` element is being swapped.

Comment (sarahmonster) [inline archive-person.php]: After all my talk of tables making sense for tabular data....I ended up swapping out the tables altogether. 😉 

I'm happy to help update the markup—it's mostly the same, just uses some extra classes (`strikebase-card`). There are a few changes in `template-tags.php` as well where I've tweaked the output, but those should be obvious and/or trivial to re-implement.

Comment (danielwrobert) [inline archive-person.php]: Yeah, I'm not too concerned about it at all. I am just noting so it's on our mind to sort out before/during the `rebase` into the React branch.

> There are a few changes in template-tags.php as well where I've tweaked the output, but those should be obvious and/or trivial to re-implement.
Also, depending on how the data is available to React, some of the template-tags will probably end up getting removed after cleaning up, once everything is sorted out in React. But not all of them. Some are still being used to modify the endpoints.

**Comment by sarahmonster** [comment]
> The Francois One font shows up as Courier on my end, but maybe that'll change once this is merged?

That shouldn't be happening—I just double-checked and I'm seeing it load from Google fonts. (FWIW I'm not 100% sold on this typeface as of yet but I'd still like to see it working properly...)

screenshot 2017-02-21 11 43 46

Can you check to see if the style is being loaded properly from the googleapis.com domain?

**Comment by michiecat** [comment]
@sarahmonster ah, so when I view the actual site, Francois One loads properly. It's only when I access the style guide via the theme's directory that I'm seeing Courier instead. Does that happen on your end?

**Comment by sarahmonster** [comment]
Got it—the typeface CSS from Google wasn't being loaded (because the style guide files don't hook into WordPress at present, although I may change this for a later iteration). Good catch! 

Fixed now.

**Comment by michiecat** [comment]
Looks good to me! Merging.

**Pull request merged**